secrets-provider/azure/README.md

```diff
@@ -0,0 +1,118 @@
+# Pulumi Azure KMS Encryption
+
+Pulumi allows you to use KMS encryption from your cloud provider to encrypt any secrets stored in the backend.
+
+This example shows how this might be done for Azure Keyvault. It creates a storage bucket with a single file that has a "secret" value.
+
+# Getting Started
+
+To use this example, perform the following steps. This examples assumes you have the pulumi-cli installed and the gcloud SDK installed.
```

Comment:
Not gcloud SDK

Reply:
Fixed! Copy paste fail, thanks for spotting!